spaces/domains.json

```
@@ -53,13 +53,16 @@
   "vote.frontier.xyz": "frontier",
   "vote.oceanprotocol.com": "oceandao",
   "vote.sav3.org": "sav3",
+  "vote.bt.finance": "btfinance.eth",
+  "vote.zlot.finance": "zlot"
   "vote.zlot.finance": "zlot",
   "vote.finnexus.io": "fnx",
   "signal.idle.finance": "idlefinance.eth",
   "gov.flush.lol": "flushlol.eth",
   "fund.emptyset.finance": "esd.eth",
   "mtrg.meter.io": "meter-mainnet.eth",
   "emtrg.meter.io": "meter-eth.eth",
+  "gov.kp2r.network": "keep2r.eth"
   "gov.kp2r.network": "keep2r.eth",
```

When you added your domain, you duplicated 2 lines in domain.json
```suggestion
  "gov.kp2r.network": "keep2r.eth",
```